cypress/integration/App.spec.js

```diff
@@ -0,0 +1,71 @@
+import '@testing-library/cypress/add-commands';
+import { getCurrentMonth } from '../../src/utils/helpers';
+
+describe('App', () => {
+  it('adds expenses', () => {
+    // ARRANGE
+    // Stub responses for getting and posting expense items
+    cy.intercept('GET', '/items*', []);
+    cy.intercept('POST', '/items*', {
+      id: 5,
+      title: 'Tasty test tacos',
+      date: '2022-02-01',
+      amount: 1499,
+    });
+
+    // ACT
+    // Visit website
+    cy.visit('/');
+
+    // Type an expense name, date, amount, and click "Add"
+    cy.findByRole('textbox', { name: /expense name/i }).type('Test Tacos');
+    cy.findByLabelText(/expense date/i).type('2022-02-01');
+    cy.findByRole('spinbutton', { name: /expense amount in €/i }).type('14.99');
```

Comment:
i would probably remove the `in €` here. but that's just me

Reply:
It is a bit redundant, but maybe it doesn't hurt to have it there since there isn't anything else in the app right now that indicates to the user that it expects euros.